packages/devkit/src/utils/binary-extensions.ts

```diff
@@ -0,0 +1,39 @@
+import { extname } from 'path';
+
+const binaryExtensions = new Set([
+  // // Image types originally from https://github.com/sindresorhus/image-type/blob/5541b6a/index.js
+  '.jpg',
+  '.jpeg',
+  '.png',
+  '.gif',
+  '.webp',
+  '.flif',
+  '.cr2',
+  '.tif',
+  '.bmp',
+  '.jxr',
+  '.psd',
+  '.ico',
+  '.bpg',
+  '.jp2',
+  '.jpm',
+  '.jpx',
+  '.heic',
+  '.cur',
+  '.tgz',
+
+  // Java files
+  '.jar',
+  '.keystore',
+
+  // Font files
+  '.ttf',
+  '.otf',
+  '.woff',
+  '.woff2',
+  '.eot',
+]);
+
+export function isBinaryFile(path: string) {
+  return binaryExtensions.has(extname(path));
```

Comment:
Do we want to list out more extensions like `.pdf`, `.gz`, `.zip`, etc.? 

There's a long list here: https://github.com/sindresorhus/binary-extensions/blob/main/binary-extensions.json

Reply:
I added these to the list.. let's see if we want to import this list differently in the future. :thinking: